app/assets/javascripts/components/shipment.js.coffee

```diff
@@ -34,7 +34,13 @@ window.Shipment = class Shipment extends React.Component
       td({}
         data.purpose_code + ' - ' + data.source_code + ' - ' + data.year
       )
-      unless @state.changesHistory
+      if @state.changesHistory
+        [
```

Comment:
I think these 2 columns at the end are a bit of a stretch for the layout. Would it work to use the "Reference" column instead? I just realised in Michel's design we had the timestamp there, which would make more sense. Not sure what to do about the updater, but maybe if the first column was a bit wider it could go there as well? e.g.
2016/12/20
1 change by WCMC